samples/subsys/ipc/ipm_imx/README.rst

```diff
@@ -0,0 +1,53 @@
+.. _ipm_imx:
+
+i.MX IPM example
+################
+
+Overview
+********
+A simple example that can be used with multicore i.MX SoC's containing the
+Messaging Unit peripheral. It reads the received data from the Messaging Unit
+using the IPM and transmits it back unchanged. The information about
+the received data is printed to the console. When sending the data back,
+it blocks until the data are read by the remote side.
+
+The i.MX Messaging Unit peripheral consists of 4 32-bit transmit and receive
+registers. The sample uses the option CONFIG_IPM_IMX_MAX_DATA_SIZE_4, which
+effectively creates 4 IPM channels. Selecting the option
+CONFIG_IPM_IMX_MAX_DATA_SIZE_8 or CONFIG_IPM_IMX_MAX_DATA_SIZE_16 would result
+in two 64-bit channels or a single 128-bit channel respectively.
+
+Note that this is just a simple demo to demonstrate the i.MX IPM functionality
+and blocking sending of the data back is done in the interrupt which would not
+be appropriate for the real world application.
+
+Building and Running
+********************
+
+The sample requires that data are being sent by the remote core so it can echo
+it back. The other core is usually not running Zephyr but Linux on i.MX
+multicore processors, this code is not part of this Zephyr sample.
+
+This project outputs data to the console.
+It can be built as follows:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/ipc/ipm_imx
+   :board: udoo_neo_full_m4
+   :goals: build flash
+   :compact:
+
+Follow the instructions from board documentation on how to load the Zephyr
```

Comment:
Change to include a reference to the board documentation:

    Follow the instructions in the :ref:`udoo_neo_full_m4` board documentation
    for how to load the Zephyr binary to the desired core and execute it.

Reply:
I have changed this, thank you.